examples/mallocMC_example01.cpp

```diff
@@ -67,7 +82,7 @@ struct ShrinkConfig
 using ScatterAllocator = mallocMC::Allocator<
     Acc,
     mallocMC::CreationPolicies::Scatter<ScatterHeapConfig, ScatterHashConfig>,
-    mallocMC::DistributionPolicies::XMallocSIMD<XMallocConfig>,
+    mallocMC::DistributionPolicies::Noop,
```

Comment:
Is this PR urgent? Because I would prefer fixing `declareSharedVar` in alpaka and not disable it temporarily.

Reply:
I will enable the usage as soon as it is fixed in alpaka, currently, mallocMC would crash with CPU accelerators.

Reply:
You mean reenable in a separate PR?

Reply:
Yes I will re-enable it in a separate PR after it is fixed in alpaka.

Reply:
Go ahead then.

Reply:
I think I would have prefered a TODO comment here, but I trust that you will change it back once the issue with `declareSharedVar` is fixed in alpaka.